### PR TITLE
[Bug Fix]LPS-116325

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test:sf

**Comment by tinatian** [comment]
FYI @vicnate5 This is fix for future ClassNotFoundException when doing rolling restart.

**Comment by yichenroy** [comment]
ci:test:relevant

CI had issues with fetching branch.

File: modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/jgroups/JGroupsReceiver.java [hunk 1]
```diff
@@ -44,6 +49,36 @@ public JGroupsReceiver(
...
+			}
+
+			String contextNamePrefix = contextName.substring(0, index + 1);
+
+			for (Map.Entry<String, ClassLoader> entry :
+					classLoaderMap.entrySet()) {
+
+				String key = entry.getKey();
+
+				if (key.startsWith(contextNamePrefix)) {
```
Comment (shuyangzhou) [inline modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/jgroups/JGroupsReceiver.java:69]: @tinatian this is not right, how do you know the first match is the highest version? And looping through the entrySet is very slow, considering how many bundles we got.

**Comment by shuyangzhou** [comment]
Waiting on the new pull.

**Pull request closed without merging**